idea/task-web-client/README.md

```diff
@@ -0,0 +1,67 @@
+# Set up for serving from HublSoft.Task.Monitor
+
+
+
+## Create and link the build
```

Comment:
Using a symlink is fairly non-standard on Windows. Known to cause problems with e.g. Visual Studio. Also because each time you build the code, the rendered filenames change (for cache-busting purposes) which then requires you to manually update your Razor view to reference the new filename.